src/components/Common/OpenPositions.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+
+import { Col, Row, Grid } from '../grid'
+import JobsByLocation from './JobsByLocation'
+import { SectionTitle, Subtitle } from '../Typography'
+import JobLink from './JobLink'
+import GreyBackground from './GreyBackground'
+
+export const OpenPositionsWithRef = React.forwardRef((props, ref) => (
+  <div ref={ref}>
+    <OpenPositions {...props} limit={40} />
```

Comment:
I would include `limit` in the props here not hard coding it to 40, in its current use we do want *all* of the jobs to be displayed but remember we're in the `Common` directory. In general any reusable components should have a flexible API.

e.g. if we wanted to add a "go to jobs" button on the homepage that would scroll down to a list of roles on the same page, we'd want to use this component but would need to rewrite the props to have a custom limit